test/dotnet-test.Tests/GivenDotnetTestBuildsAndRunsTestFromCsprojForMultipleTFM.cs

```diff
@@ -75,5 +76,23 @@ public void XunitMultiTFM()
             result.StdOut.Should().Contain("Failed   TestNamespace.VSTestXunitTests.VSTestXunitFailTestNetCoreApp");
             result.ExitCode.Should().Be(1);
         }
+
+        [Fact]
+        public void ItCanTestAMultiTFMProjectWithImplicitRestore()
+        {
+            var testInstance = TestAssets.Get(
+                    TestAssetKinds.DesktopTestProjects,
+                    "MultiTFMXunitProject")
+                .CreateInstance()
+                .WithSourceFiles();
+
+            string projectDirectory = Path.Combine(testInstance.Root.FullName, "XUnitProject");
+
+            CommandResult result = new DotnetTestCommand()
+                       .WithWorkingDirectory(projectDirectory)
+                       .ExecuteWithCapturedOutput($"{TestBase.ConsoleLoggerOutputNormal} --framework netcoreapp2.0");
+
+            result.StdOut.Should().Contain("Total tests: 2. Passed: 1. Failed: 1. Skipped: 0.");
```

Comment:
Also check that exit code. We won't be able to check just the string in loc case. For our own strings, I am checking against the actual resources, but for strings from VSTest, we can't do that so I'm conditioning these checks. There should always be a sanity check of the exit code when these are not checked on non-English.